utils/test/Is_LSMB_running.sh

```diff
@@ -13,6 +13,20 @@ if [[ -z $EX_NOHOST ]]; then
     exit 99;
 fi
 
+
+# base url for lsmb defaults to http://localhost:5001
+# this can be overriden by setting environment variable $LSMB_BASE_URL
+lsmb_base_url=${LSMB_BASE_URL:-http://localhost:5001}
+
+# We check Starman is running on a specific port
+url_without_port=${lsmb_base_url%:[[:digit:]]*}
+lsmb_port=${lsmb_base_url#$url_without_port}
+lsmb_port=${lsmb_port#:}  # strip colon
```

Comment:
If you insert the colon before the $ on line 23 it will work better and line 24 can be dropped

Reply:
Sorry, that's not clear, and puts the : at the wrong end of the variable.
An Example is probably better and this one is tested to be correct
lsmb_port=${lsmb_base_url#${url_without_port}:}

Reply:
I'm grateful for the feedback on this, as bash string matching is a foreign language for me - I've dusted off the phrase book!

But, the change you suggest breaks if LSMB_BASE_URL omits the port, for example:
LSMB_BASE_URL=http://localhost

The version I coded, while uglier, uses the default port in that situation.